web/src/app/schedules/temp-sched/TempSchedShiftsList.tsx

```diff
@@ -57,33 +71,99 @@ export default function TempSchedShiftsList({
-
     const displaySpan = Interval.fromDateTimes(
-      DateTime.min(schedInterval.start, firstShiftStart).startOf('day'),
-      DateTime.max(schedInterval.end, lastShiftEnd).endOf('day'),
+      DateTime.fromISO(start).startOf('day'),
+      DateTime.fromISO(end).endOf('day'),
     )
 
     const result: FlatListListItem[] = []
-    displaySpan.splitBy({ days: 1 }).forEach((day, idx) => {
-      const dayShifts = shifts.filter((s) => day.overlaps(s.interval))
-      if (!dayShifts.length) return
+    const days = displaySpan.splitBy({ days: 1 })
+    days.forEach((day, dayIdx) => {
+      const dayShifts = sortedShifts.filter((s) => day.overlaps(s.interval))
+
+      // if no shifts, only render subheaders for start/end days
+      if (!sortedShifts.length && dayIdx > 0 && dayIdx < days.length - 1) {
+        return
+      }
 
       // render subheader for each day
       result.push({
         subHeader: relativeDate(day.start),
       })
 
-      // craft user friendly shift string
-      dayShifts.forEach((s) => {
+      // add start time of temp schedule to top of list
+      if (dayIdx === 0) {
+        result.push({
+          render: () => (
+            <Alert
+              key='start'
+              className={classes.alert}
+              severity='success'
+              icon={<ScheduleIcon />}
+            >
+              Starts at{' '}
+              {DateTime.fromISO(start).setZone(zone).toFormat('h:mm a')}
+            </Alert>
+          ),
+        })
+
+        // render no coverage/get started below start time if no shifts
+        if (!sortedShifts.length) {
+          result.push({
+            render: () => (
+              <Alert
+                key='no-coverage'
+                className={classes.alert}
+                severity='info'
+              >
+                <AlertTitle>No coverage</AlertTitle>
+                Add a shift to get started
+              </Alert>
+            ),
+          })
+        }
+      }
+
+      // for temp scheds with at least 1 shift
+      // render no coverage and continue if no shifts for the given day
+      if (!dayShifts.length && sortedShifts.length) {
+        return result.push({
+          render: () => (
+            <Alert
+              key={day.start.toISO() + '-no-coverage'}
+              className={classes.alert}
+              severity='warning'
+            >
+              No coverage
+            </Alert>
+          ),
+        })
+      }
+
+      // checkCoverage will determine if there is a gap of 1 minute or more between the given datetimes
+      const checkCoverage = (s: DateTime, e: DateTime): boolean => {
+        return Interval.fromDateTimes(s, e).length('minutes') > 1
+      }
+
+      // craft list items for each day
+      dayShifts.forEach((s, shiftIdx) => {
+        // check start of day coverage for the first shift
+        // if on the first day, temp sched start is used
+        const _s = dayIdx === 0 ? DateTime.fromISO(start) : day.start
```

Comment:
`dayShifts` is evaluated on every render, so it will have the most recent day index and start of schedule here if they change the start and come back.